packages/blockchain-wallet-v4-frontend/src/store/index.js

```diff
@@ -90,9 +91,20 @@ const configureStore = async function () {
   })
   const persistWhitelist = ['session', 'preferences', 'cache']
 
-  // TODO: remove getStoredStateMigrateV4 someday (at least a year from now)
-  const store = createStore(
-    connectRouter(history)(
+  const store = configureStore({
```

Comment:
I just did a basic amount of effort here to get RTK working, but there's more we could do here. Converting to TS file, and using RTKs defaultMiddleware are a couple wins.